packages/core/src/components/spinner/spinner.tsx

```diff
@@ -0,0 +1,18 @@
+import { Component, h, Host, Prop } from "@stencil/core";
+
+@Component({
+  tag: "fl-spinner",
+  styleUrl: "spinner.scss",
+  shadow: true,
+})
+export class Spinner {
+  @Prop() size: "sm" | "md" | "lg" = "md";
+
+  render() {
+    return (
+      <Host>
```

Comment:
Might be missing something, but we don't need to use `Host` here right?

Reply:
Makes setting the props a lot easier for some reason.